docs/core/tools/dotnet-build.md

```diff
@@ -1,20 +1,20 @@
 ---
 title: dotnet build command
 description: The dotnet build command builds a project and all of its dependencies.
-ms.date: 11/27/2023
+ms.date: 09/24/2025
 ---
 # dotnet build
 
-**This article applies to:** ✔️ .NET Core 3.1 SDK and later versions
+**This article applies to:** ✔️ .NET 6 and later versions
 
 ## Name
 
-`dotnet build` - Builds a project and all of its dependencies.
+`dotnet build` - Builds a project, solution, or file-based program and all of its dependencies.
```

Comment:
global change:  Instead of "file-based program", the product name is "file-based app". (I think I need to change some as well.)